modules/admin_manual/pages/installation/docker/index.adoc

```diff
@@ -125,7 +125,7 @@ stored in `.env` earlier.
 
 Assuming you used docker-compose, as in the previous example, to stop
 the containers use `docker-compose stop`. Alternatively, use
-`docker-compose down` to stop and remove containers, along with the
+`docker-compose down --rmi all --volumes` to stop and remove containers, along with the
```

Comment:
I'm doubting it would, but would it also be worth also including `--remove-orphans`? Otherwise, looks good.